.goreleaser.yml

```diff
@@ -8,9 +8,14 @@ builds:
       - linux
```

Comment:
It looks like Terraform also recommends to build for `freebsd`. Let's add it:
```suggestion
      - freebsd
      - linux
```

Reply:
Nice catch. Thanks :+1: